src/norm/sqlite.nim

```diff
@@ -124,7 +130,7 @@ proc createTables*[T: Model](dbConn; obj: T) =
 
 # Row manipulation
 
-proc insert*[T: Model](dbConn; obj: var T, force = false, conflictPolicy = cpRaise) =
+proc insert*[T: Model](dbConn; obj: var T; force = false; conflictPolicy = cpRaise) =
```

Comment:
@PhilippMDoerner are you using an autoformatter? I'm not really against semicolons but I'd rather have one style across the codebase than two ones :-)

Could you please replace semicolons with commas?

Reply:
Ah yes, I have nimpretty set up to apply automatically. I don't know why it does that, I'll fix it

Reply:
@moigagoo not my PR :laughing:

Reply:
@PhilippMDoerner Oops, sorry! 😬